### PR TITLE
Languages2

### PR DESCRIPTION
**Pull request by petzlux**

### CODE REVIEW

**Comment by petzlux** [comment]
Jquery doesnt get downloaded during build ? Anyone know why ?

**Comment by elemoine** [comment]
> Jquery doesnt get downloaded during build ? Anyone know why ? @elemoine

With the master branch jQuery does get downloaded for me when I run the `build` target.

**Comment by petzlux** [comment]
@sbrunner Jeff me disait que t'avais des commentaires concernant les changements de CONST_Makefile ?

**Comment by sbrunner** [comment]
Hello,

Yes, some task like `update-translation` is luxembourg specific than it shouldn't be in the `CONST_Makefile`, but actually we can't move it properly to the `geoportailv3.mk`, then I will move it later :-)

**Comment by petzlux** [comment]
Ready for merge ?

File: geoportailv3/utils/db_to_pot.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from os import path
+import codecs
+
+
+pathToPotFiles = './geoportailv3/locale/'
+p = path.join(pathToPotFiles, "geoportailv3-db.pot")
+
+if __name__ == "__main__":
+    w = codecs.open(p, 'w', encoding='utf-8')
+    w.write('#, fuzzy\nmsgid ""\nmsgstr ""\n"Plural-Forms: nplurals=2; \
```
Comment (sbrunner) [inline geoportailv3/utils/db_to_pot.py]: for me the 
```
#, fuzzy
msgid ""
msgstr ""
```
looks strange Is it relay needed ?

**Comment by sbrunner** [comment]
Just a last little question, otherwise that's looks good :-)

CU
Stéph

File: geoportailv3/utils/db_to_pot.py (hunk 1 above)
Comment (petzlux) [inline geoportailv3/utils/db_to_pot.py]: AFAIK, it is part of the specification of a pot / po file, and gettext tools will fail if not present.

Comment (sbrunner) [inline geoportailv3/utils/db_to_pot.py]: OK, thanks for the verification :-)

**Comment by petzlux** [comment]
If I want to merge into master, than will Pierrre's Commits here appear twice in master ? It looks this way in my git log at the moment , so I dont want to do a push

**Comment by elemoine** [comment]
Sounds like some rebase is needed. I'll look at it later today or tomorrow morning if you don't feel comfortable with the required operation.

**Comment by petzlux** [comment]
after a couple of unsuccessful tries, #149 seems to look better and can be merged without conflicts !

**Pull request closed without merging**